src/clj/harja/kyselyt/tarkastukset.sql

```diff
@@ -920,3 +920,27 @@ WHERE id = :id;
 
 --name: tarkastuksen-urakka
 SELECT urakka FROM tarkastus WHERE id = :id;
+
+-- name: hae-urakan-ei-kayty-tieturvallisuusverkko
+WITH tieturvallisuustarkastukset AS
+         (SELECT ST_UNION(t.sijainti) AS sijainti
+          FROM tarkastus t
+          WHERE t.urakka = :urakka
+            AND t.aika BETWEEN :alku::TIMESTAMP AND :loppu::TIMESTAMP
+            AND t.tyyppi = 'tieturvallisuus'
```

Comment:
Melkein voisi lisätä vielä tyypin tuohon tyyppiin eli 
AND t.tyyppi = 'tieturvallisuus'::tarkastustyyppi